controls/pcidss_4.yml

```diff
@@ -2045,1274 +2629,1420 @@ controls:
+        prodtypes to avoid conflicts.
       rules:
-      - var_multiple_time_servers=suse
-      - ntpd_specify_remote_server
-      - ntpd_specify_multiple_servers
-      - chronyd_specify_remote_server
-      - chronyd_or_ntpd_specify_multiple_servers
+        # specific products should update the variable value when defining the profile based on
+        # this control file.
+        - var_multiple_time_servers=generic
+        - chronyd_specify_remote_server
+        - chronyd_or_ntpd_specify_multiple_servers
+        - ntpd_specify_remote_server
+        - ntpd_specify_multiple_servers
+      related_rules:
+        - chronyd_configure_pool_and_server
 
     - id: 10.6.3
-      title: 'Time synchronization settings and data are protected'
+      title: Time synchronization settings and data are protected.
+      description: |-
+        - Access to time data is restricted to only personnel with a business need.
+        - Any changes to time settings on critical systems are logged, monitored, and reviewed.
       levels:
         - base
       status: automated
       rules:
-      - audit_rules_time_watch_localtime
-      - audit_rules_time_watch_localtime
-      - audit_rules_time_settimeofday
-      - audit_rules_time_clock_settime
-      - audit_rules_time_stime
-      - audit_rules_time_adjtimex
+        - audit_rules_time_watch_localtime
+        - audit_rules_time_watch_localtime
+        - audit_rules_time_settimeofday
+        - audit_rules_time_clock_settime
+        - audit_rules_time_stime
+        - audit_rules_time_adjtimex
+        - chronyd_run_as_chrony_user
 
-  - id: 10.7
-    title: 'Failures of critical security control systems are detected, reported, and
-      responded to promptly.'
+  - id: '10.7'
+    title: Failures of critical security control systems are detected, reported, and responded to
+      promptly.
     levels:
       - base
     status: partial
-    rules:
-    - grub2_audit_argument
     controls:
     - id: 10.7.1
-      title: 'Additional requirement for service providers only: Failures of critical
-        security control systems are detected, alerted, and addressed promptly,
-        including but not limited to failure of the following critical security control
-        systems'
-      levels:
-        - base
-      status: pending
-      rules: []
+      title: 'Additional requirement for service providers only: Failures of critical security
+        control systems are detected, alerted, and addressed promptly.'
+      description: |-
+        It includes but is not limited to failure of the following critical security control
+        systems:
+        - Network security controls.
+        - IDS/IPS.
+        - FIM.
+        - Anti-malware solutions.
+        - Physical access controls.
+        - Logical access controls.
+        - Audit logging mechanisms.
+        - Segmentation controls (if used).
+      levels:
+        - base
+      status: manual
```

Comment:
It is not mandated to send warnings by email. The policy is flexible regarding solutions used to warn the admins. For example, if a SIEM solution is used, it might not be necessary to configure components individually. So, I don't think we should include these rules. We have rules to configure what happens if the storage is full in order to preserve logs, but how to warn it depends on site policy.